{
	"vulinfoid": "JVNDB-2018-011024",
	"vulinfodata": {
		"title": "libtirpc におけるリソースの枯渇に関する脆弱性",
		"vulinfodescription": {
			"overview": "libtirpc には、リソースの枯渇に関する脆弱性が存在します。"
		},
		"affected": {
			"affecteditem": [
				{
					"name": "libtirpc project",
					"productname": "LIBTIRPC",
					"cpe": {
						"text": "cpe:/a:libtirpc_project:libtirpc",
						"version": "2.2"
					},
					"versionnumber": [
						"1.0.2-rc2 未満"
					]
				}
			]
		},
		"impact": {
			"cvss": [
				{
					"version": "2.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.8",
					"vector": "AV:N/AC:L/Au:N/C:N/I:N/A:C"
				},
				{
					"version": "3.0",
					"severity": {
						"text": "High",
						"type": "Base"
					},
					"base": "7.5",
					"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"impactitem": {
				"description": "サービス運用妨害 (DoS) 状態にされる可能性があります。"
			}
		},
		"solution": {
			"solutionitem": {
				"description": "ベンダ情報および参考情報を参照して適切な対策を実施してください。"
			}
		},
		"related": {
			"relateditem": [
				{
					"type": "vendor",
					"name": "SourceForge",
					"vulinfoid": "libtirpc",
					"url": "https://sourceforge.net/projects/libtirpc/"
				},
				{
					"type": "advisory",
					"name": "Common Vulnerabilities and Exposures (CVE)",
					"vulinfoid": "CVE-2018-14621",
					"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-14621"
				},
				{
					"type": "advisory",
					"name": "National Vulnerability Database (NVD)",
					"vulinfoid": "CVE-2018-14621",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-14621"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "Bug 1620290",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14621"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "rendezvous_request: fix endless loop in EMFILE case",
					"url": "http://git.linux-nfs.org/?p=steved/libtirpc.git;a=commit;h=fce98161d9815ea016855d9f00274276452c2c4b"
				},
				{
					"type": "advisory",
					"name": "関連文書",
					"vulinfoid": "Bug 968175",
					"url": "https://bugzilla.novell.com/show_bug.cgi?id=968175"
				},
				{
					"type": "cwe",
					"name": "JVNDB",
					"vulinfoid": "CWE-400",
					"url": "https://cwe.mitre.org/data/definitions/400.html",
					"title": "リソースの枯渇"
				}
			]
		},
		"history": {
			"historyitem": [
				{
					"historyno": "1",
					"datetime": "2019-01-04T15:45:20+09:00",
					"description": "[2019年01月04日]\\n  掲載"
				}
			]
		},
		"datefirstpublished": "2019-01-04T15:45:20+09:00",
		"datelastupdated": "2019-01-04T15:45:20+09:00",
		"datepublic": "2018-08-22T00:00:00+09:00"
	}
}
